.changeset/hip-tips-lay.md

```diff
@@ -0,0 +1,7 @@
+---
+"@pnpm/filter-workspace-packages": major
+"pnpm": major
```

Comment:
this should be probably minor not major

major will bump pnpm to v6

Reply:
done [here](https://github.com/pnpm/pnpm/pull/3035/commits/02de2a59eea4b246c8eb1afb9feda15956cba2dc)